### PR TITLE
DPC-1174: Fix headers in webapp

### PR DESCRIPTION
**Pull request by Sun-Mountain**
### Fixes [DPC-1174](https://jira.cms.gov/browse/DPC-1174)

Currently the headers (Home, Documentation, Frequently Asked Questions) in the webapp do not have the same formatting as the static site.

### Proposed Changes

The long-story-short reason why this is happening: the way Jekyll compiles CSS is slightly different than Rails, hence the small but noticeable differences.

### Change Details

I adjusted the variables to make it a cleaner transition.

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### Acceptance Validation

**Before**
Screen Shot 2021-03-10 at 11 39 18 AM

Screen Shot 2021-03-10 at 11 39 43 AM

**After**
Screen Shot 2021-03-10 at 11 57 00 AM

Screen Shot 2021-03-10 at 11 57 29 AM

### CODE REVIEW

File: _sass/components/_navbar.scss [hunk 1]
```diff
@@ -4,18 +4,20 @@
...
+  font-size: 21px;
+  font-stretch: ultra-condensed;
   font-weight: $font-bold;
+  letter-spacing: 0.1px;
   text-decoration: none;
   width: 15rem;
 
   @media (min-width: $width-sm) {
-    font-size: $h3-font-size;
+    // font-size: 12px;
```
Comment (PatKelsh) [inline _sass/components/_navbar.scss:15]: if we don't need this, can we delete it?

**Review by PatKelsh — CHANGES_REQUESTED** [review]
Just one small change

File: _sass/components/_navbar.scss (hunk 1 above)
Comment (Sun-Mountain) [reply _sass/components/_navbar.scss:15]: Yes we can

Comment (Sun-Mountain) [reply _sass/components/_navbar.scss]: Removed!

**Review by PatKelsh — APPROVED** [review]

**Pull request merged**